style.css

```diff
@@ -99,6 +99,7 @@ a:active {
 body > .is-root-container,
 .edit-post-visual-editor__post-title-wrapper,
 .wp-block-group.alignfull,
+.wp-block-group.has-background,
```

Comment:
This overwrites the editor's (random) left/right padding rules for group blocks that have a background color, so that content in there will line up with our usual site padding rules. 

Before|After
---|---
<img width="614" alt="Screen Shot 2022-01-13 at 8 54 22 AM" src="https://user-images.githubusercontent.com/1202812/149342821-a728c4fb-7d5f-46cd-a991-aa6c77272b36.png">|<img width="614" alt="Screen Shot 2022-01-13 at 8 54 04 AM" src="https://user-images.githubusercontent.com/1202812/149342840-4534c58b-b2ee-46c3-97e9-d99337ea6aa7.png">